types/yargs/yargs-tests.ts

```diff
@@ -305,6 +305,7 @@ function Argv$command() {
                 console.dir(argv.url);
             }
         )
+        .example([['get', 'get https://example.com']], '')
```

Comment:
This doesn't match the documentation.  http://yargs.js.org/docs/#api-reference-examplecmd1-desc1-cmd2-desc2

It's not completely obvious but the functions named `Argv$foo()` test feature foo.  This is in `function Argv$command()` which is meant to test `.command()`.  Instead it should be in a new `function Argv$example()`.